sdk/test/Services/S3/UnitTests/Custom/MultipartDownloadManagerTests.cs

```diff
@@ -2044,16 +1976,22 @@ public async Task MultipleDownloads_WithSharedHttpThrottler_RespectsLimits()
             var coordinator1 = new MultipartDownloadManager(mockClient1.Object, request1, config, mockDataHandler1.Object, null, sharedThrottler);
             var coordinator2 = new MultipartDownloadManager(mockClient2.Object, request2, config, mockDataHandler2.Object, null, sharedThrottler);
 
-            // Act - Start both discoveries concurrently
-            var task1 = coordinator1.DiscoverDownloadStrategyAsync(CancellationToken.None);
-            var task2 = coordinator2.DiscoverDownloadStrategyAsync(CancellationToken.None);
+            var discovery1 = await coordinator1.DiscoverDownloadStrategyAsync(CancellationToken.None);
```

Comment:
had to update these because now with new change the semaphore is held through start download, so callers need to do discover + start download in order to get through the whole http cycle